torch/_inductor/codegen/cpp.py

```diff
@@ -455,6 +455,10 @@ def mul(a, b):
     def div(a, b):
         return f"{a} / {b}"
 
+    @staticmethod
+    def truediv(a, b):
+        return f"{a} / {b}"
```

Comment:
Will this be correct for integer types?

In C++ int/int is floordiv, while float/float is truediv.

Add a test case for this.

Reply:
Integer types don't hit it. I added a test case for int div with a TODO to vectorize them.